code/src/Style/ThoughtSubmitForm.css

```diff
@@ -0,0 +1,56 @@
+.button-submit-thought {
+  background-color: #8a5a44;
+  border-radius: 20px;
+  border: none;
+  color: #ffffff;
+  display: flex;
+  align-items: center;
+  font-family:  'Capriola', sans-serif;
+  font-size: 13px;
+  margin-top: 10px;
+  padding: 5px 15px;
+  
+}
+
+.thought-box {
+  height: 45px;
+  margin-top: 20px;
+  font-size: 18px;
+}
+.thought-form-label {
+  color: rgb(2, 2, 2);
+  font-family: 'Capriola', sans-serif;
+}
+
+.error-display {
+  align-items: flex-start;
+  display: flex;
+}
+
+.error-message {
+  font-style: italic;
+  font-size: 13px;
+}
+
+.word-count {
+  color: blue;
+  flex: 1;
+  font-family: 'Raleway', sans-serif;
+  font-weight: 500;
+  font-size: 13px;
+  text-align: end;
+}
```

Comment:
I think you could be more consistent using blank spaces inbetween your css-classes - either no spaces or one space.